src/main/java/com/sourcegraph/cody/MyAccountTabPanel.kt

```diff
@@ -79,7 +81,16 @@ class MyAccountTabPanel(val project: Project) : JPanel() {
             button("Upgrade") { BrowserUtil.browse(ConfigUtil.DOTCOM_URL + "cody/subscription") }
         upgradeButton.component.putClientProperty(DarculaButtonUI.DEFAULT_STYLE_KEY, true)
       }
-      button("Check Usage") { BrowserUtil.browse(ConfigUtil.DOTCOM_URL + "cody/manage") }
+      button("Manage Account") {
+        val username =
+            CodyAuthenticationManager.getInstance(project)
+                .getActiveAccount()
+                ?.let(CodyAccount::name)
```

Comment:
You should be able to chain the access:

```suggestion
                ?.name
```

The suggestion is equivalent to `x?.name` in TypeScript. The thing you wrote is a bit like `x ? Object.getOwnPropertyDescriptor(CodyAccount.prototype, 'name').get.apply(x) : null`.

Reply:
lol. Fixed!